apiservers/portlayer/restapi/handlers/exec_handlers.go

```diff
@@ -122,9 +181,23 @@ func (handler *ExecHandlersImpl) ContainerCreateHandler(params exec.ContainerCre
 				Cmd: cmd,
 			},
 		},
+		Networks: make(map[string]metadata.NetworkEndpoint),
 	}
 	log.Infof("Metadata: %#v", m)
 
+	// network config
+	ns := params.CreateConfig.NetworkSettings
+	ne, s, err := addContainerToScope(name, ns)
+	defer func() {
+		if err != nil && s != nil {
```

Comment:
This looks like a default error handler for this method.  You might want to log that an error is being caught here.